CHANGELOG.rst

```diff
@@ -7,8 +7,27 @@ All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
 
-[Unreleased 1.1.9] - `master`_
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+[Unreleased 1.1.10] - `master`_
```

Comment:
```suggestion
[Unreleased 1.2.0] - `master`_
```

Reply:
is the next release going to be a minor then?

Reply:
just continuing how we have done it before when went above `9`. guess a `1.1.10` could be a bit confusing vs `1.1.1`

Reply:
thought we always just put it as the next iteration in the major.minor.patch format

Reply:
yep, but `.10` is the same as `.1`. feel free to try whether everything works then :-D

Reply:
Ohhh. Didn't even think about it as "one zero"... @tmbo do we have a specific protocol for going above patch 9?

Reply:
From https://semver.org/#semantic-versioning-specification-semver:

> A normal version number MUST take the form X.Y.Z where X, Y, and Z are non-negative integers, and MUST NOT contain leading zeroes. X is the major version, Y is the minor version, and Z is the patch version. Each element MUST increase numerically. For instance: 1.9.0 -> 1.10.0 -> 1.11.0.

Reply:
@tmbo

Reply:
this should be a feature release (as it includes new features, e.g. sending messages to the user with the route tobias added or the new command line flag).

in general though, `1.1.10` will work just fine as a patch release after `1.1.9`, there is no issue with using `10`

Reply:
Okay. so 1.2.0 instead of 1.1.9? I feel like we've been bad about separating features and patches since the rasa 1.0 release (we still don't have a .x branch)